applications/nrf_desktop/src/modules/ble_bond.c

```diff
@@ -292,7 +309,13 @@ static int shell_show_peers(const struct shell *shell, size_t argc, char **argv)
 static int shell_remove_peers(const struct shell *shell, size_t argc,
 			      char **argv)
 {
-	return remove_peers(cur_peer_id);
+	if ((state != STATE_DONGLE) && (state != STATE_DONGLE_STANDBY) &&
+	(state != STATE_DONGLE_ERASE_PEER) && (state != STATE_DONGLE_ERASE_ADV)) {
+		return remove_peers(cur_peer_id);
+	}
+	LOG_WRN("Failed to remove peers. Change to other state from dongle state.");
+	return 0;
+
```

Comment:
remove extra empty line

Reply:
Consider using if ... else in case where we choose from 2 actions that exclude each other.

Reply:
Also maybe we could print the shell command response on shell ?